core/state.go

```diff
@@ -13,78 +13,80 @@ import (
 
 // Operators
 
-// OperatorSocket is formatted as "host:dispersalPort;retrievalPort;v2DispersalPort"
-type OperatorSocket string
+// OperatorSocket is formatted as "host:dispersalPort;retrievalPort;v2DispersalPort;v2RetrievalPort"
```

Comment:
This comment is not quite true: when v2 ports are empty, it's not in this format